tests/test_pac_bio_experiment.py

```diff
@@ -0,0 +1,106 @@
+import pytest
+from ml_warehouse.schema import PacBioRun
+from sqlalchemy import select
+
+from lang_qc.models.pacbio.experiment import PacBioExperiment
+from tests.conftest import insert_from_yaml
+
+
+def test_query(mlwhdb_test_session):
```

Comment:
You seem to be testing `from_orm` implementations here, rather than queries. Rename?

Reply:
Sorry, forgot to change teh test name after initial experiments with fixtures